pyproject.toml

```diff
@@ -55,3 +55,10 @@ django = ["django"]
 
 [tool.ruff.lint.mccabe]
 max-complexity = 20
+
+[tool.uv]
+dev-dependencies = [
```

Comment:
these should be optional dependencies named "dev", as in

https://github.com/realsuayip/asu/blob/8a4eb841018cdd702b805994117715995c98ea4f/pyproject.toml#L36